translations/translations.json

```diff
@@ -8633,7 +8659,9 @@
       "trackingLeaveDescription": "Du hast das Tracking nicht erlaubt. Das Tracking hilft uns, die {{appName}}-App zu verbessern und bessere Informationen für Einwanderer bereitzustellen. Bist du sicher, dass du den Bildschirm verlassen möchten, ohne das Tracking zu erlauben?",
       "enabled": "aktiviert",
       "disabled": "deaktiviert",
-      "decline": "Ablehnen"
+      "decline": "Ablehnen",
+      "externalResourcesTitle": "Externe Quellen",
+      "externalResourcesDescription": "Legen sie fest welche externen Quellen zugelassen werden"
```

Comment:
```suggestion
      "externalResourcesDescription": "Legen sie fest, welche externen Quellen zugelassen werden."
```

Reply:
I think we should stick to the informal you that we're using in the other translations.